sources/LLVMSharp/DataLayout.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft and Contributors. All rights reserved. Licensed under the University of Illinois/NCSA Open Source License. See LICENSE.txt in the project root for license information.
+
+using System;
+using LLVMSharp.Interop;
+
+namespace LLVMSharp
+{
+    public sealed class DataLayout : IEquatable<DataLayout>
+    {
+        public DataLayout(ReadOnlySpan<char> stringRep)
+        {
+            Handle = LLVMTargetDataRef.FromStringRepresentation(stringRep);
+        }
+
+        public LLVMTargetDataRef Handle { get; }
+
+        public StructLayout CreateStructLayout(StructType structType)
```

Comment:
```suggestion
        public StructLayout GetStructLayout(StructType structType)
```

To match the C++ name: https://llvm.org/doxygen/classllvm_1_1DataLayout.html#a61135fb8666f0b7a37b4e1bbcf1db131